### PR TITLE
[ERSSUP-59881]-[]-[Appointment Non-attendance reason refdata (i.e. DN…

### PR DESCRIPTION
**Pull request by jamiegreen4**
…A reasons)]-[JG]

## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/components/schemas/codings/AppointmentNonAttendanceReason.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+title: Appointment non-attendance reason
```
Comment (jamiegreen4) [inline specification/components/schemas/codings/AppointmentNonAttendanceReason.yaml:1]: Is this needed?

Comment (jamiegreen4) [reply specification/components/schemas/codings/AppointmentNonAttendanceReason.yaml]: Not needed

File: sandbox/src/mocks/getCodeSystem/responses/AppointmentNonAttendanceReasonCodeSystem.json [hunk 2]
```diff
@@ -0,0 +1,115 @@
+{
+    "id": "APPOINTMENT-NON-ATTENDANCE-REASON",
```
[the remark this answers is not in the document]
Comment (jamiegreen4) [reply sandbox/src/mocks/getCodeSystem/responses/AppointmentNonAttendanceReasonCodeSystem.json]: Needs to change to match what was agreed with IOPS

File: specification/components/schemas/CodeSystemConcept.yaml [hunk 3]
```diff
@@ -35,7 +35,7 @@ properties:
       | Clinic Type                     | General Urology               |
       | Appointment Cancellation Reason | Inappropriate service         |
       | Referral Cancellation Reason    | Intend to go private          |
-      | Appointment Non-attendance      | Patient ill                   |
+      | Appointment Non-attendance Reason     | Patient ill                   |
```
Comment (nhsd-david-wass) [inline specification/components/schemas/CodeSystemConcept.yaml:39]: I know it renders correctly, but just for neatness, could you align these

Comment (jamiegreen4) [reply specification/components/schemas/CodeSystemConcept.yaml:39]: I've still got branch checked out so will align

**Review by francisco-videira-nhs — APPROVED** [review]
Approved

**Pull request merged**